test/acceptance/features/steps/service_binding_request.py

```diff
@@ -4,9 +4,12 @@
 
 
 class ServiceBindingRequest(object):
-
+    namespace = ""
```

Comment:
is it used somewhere in this class later on?

Reply:
not used. removed for now. Thanks